server/app/uk/gov/ons/addressIndex/server/modules/AddressParserModule.scala

```diff
@@ -0,0 +1,18 @@
+package uk.gov.ons.addressIndex.server.modules
+
+import javax.inject.Singleton
+import uk.gov.ons.addressIndex.crfscala.CrfScala.CrfTokenResult
+import uk.gov.ons.addressIndex.parsers.AddressParser
+
+@Singleton
+class AddressParserModule {
```

Comment:
http://stackoverflow.com/questions/37282792/why-use-singleton-over-scalas-object-in-play-framework